packages/cactus-plugin-ledger-connector-cdl-socketio/src/main/typescript/connector/cdl-request.ts

```diff
@@ -0,0 +1,82 @@
+import { read as configRead } from "../common/core/config";
+import axios from "axios";
+import https from "https";
+
+// Log settings
+import { getLogger } from "log4js";
+import { readFileSync } from "fs";
+const logger = getLogger("cdl-request[" + process.pid + "]");
+logger.level = configRead("logLevel", "info");
+
+function getHttpsAgent() {
+  const agentOptions: https.AgentOptions = {};
+
+  const skipCertCheck = configRead<boolean>(
+    "cdlApiGateway.skipCertCheck",
+    false,
+  );
+  if (skipCertCheck) {
+    logger.info(
+      `Allowing self signed CDL API GW certificates (skipCertCheck=${skipCertCheck})`,
+    );
+    agentOptions.rejectUnauthorized = false;
```

Comment:
It's controlled by the user (must provide `skipCertCheck` flag)

Reply:
@outSH I agree, dismissed the alert as false-positive. 
With that said, I'd change it to assign it like 
```typescript
agentOptions.rejectUnauthorized = skipCertCheck;
```
so that if someone moves around that line of code in the future it does not accidentally make it unconditional (junior people with limited git knowledge resolving conflicts and messing up curly brace scoping for example).
It sounds a bit paranoid but definitely could happen IMO. Just a comment, not an actual change request because it's such a minor nit-pick.

Reply:
@petermetz Sure thing - fixed